src/components/modules/Projects/Projects.js

```diff
@@ -1,14 +1,35 @@
 import Link from "next/link";
+import { useState } from "react";
 import { useRouter } from "next/router";
 import Card from "../../common/Card";
 import { Layout, ProjectContainer } from "./StyledProjects";
+import SearchBar from "./Project/SearchBar";
+import Fuse from "fuse.js";
 
 const Projects = ({ projects }) => {
+  const [searchResult, setSearchResult] = useState(projects);
+
+  const options = {
+    includeScore: true,
+    keys: ["keywords.keyword"],
+    threshold: 0.3
+  };
+
+  const fuse = new Fuse(projects, options);
+
+  const searchProject = (searchQuery) => {
+    const result = fuse.search(searchQuery).map(({ item }) => item);
+    setSearchResult(result);
+  };
+
```

Comment:
is it possible to move the search functionality to the `<SearchBar />`? if not then fine we can have it here :D

Reply:
As far as I understand, because how the library works, I had to lift up the state. So, I guess It cannot be placed in the searchBar. Do you think we can get the result array and then, pass it up to the Projects file?